tests/ha/test_control_pod_restart.py

```diff
@@ -0,0 +1,411 @@
+                        " node", self.node_name, HA_CFG["common_params"]["pod_joinback_time"])
+            time.sleep(HA_CFG["common_params"]["pod_joinback_time"])
+        # TODO: As control node is restarted, Need to redeploy cluster after every test (We may
+        #  need this after control pod deployment file is changed)
+        if self.deploy:
+            LOGGER.info("Cleanup: Destroying the cluster ")
+            resp = self.deploy_lc_obj.destroy_setup(self.node_master_list[0],
+                                                    self.node_worker_list,
+                                                    const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Cleanup: Cluster destroyed successfully")
+
+            LOGGER.info("Cleanup: Setting prerequisite")
+            self.deploy_lc_obj.execute_prereq_cortx(self.node_master_list[0],
+                                                    const.K8S_SCRIPTS_PATH,
+                                                    const.K8S_PRE_DISK)
+            for node in self.node_worker_list:
+                self.deploy_lc_obj.execute_prereq_cortx(node, const.K8S_SCRIPTS_PATH,
+                                                        const.K8S_PRE_DISK)
+            LOGGER.info("Cleanup: Prerequisite set successfully")
+
+            LOGGER.info("Cleanup: Deploying the Cluster")
+            resp_cls = self.deploy_lc_obj.deploy_cluster(self.node_master_list[0],
+                                                         const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp_cls[0], resp_cls[1])
+            LOGGER.info("Cleanup: Cluster deployment successfully")
+
+        LOGGER.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.poll_cluster_status(self.node_master_list[0])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Cleanup: Cluster status checked successfully")
+
+        LOGGER.info("Removing extra files")
+        sysutils.remove_dirs(self.test_dir_path)
+        sysutils.remove_file(self.modified_yaml)
+        self.node_master_list[0].remove_remote_file(self.modified_yaml)
+        self.node_master_list[0].remove_remote_file(self.backup_yaml)
+        LOGGER.info("Done: Teardown completed.")
+
+    # pylint: disable=too-many-statements
+    # pylint: disable-msg=too-many-locals
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-32459")
+    @CTFailOn(error_handler)
+    def test_restart_control_node(self):
+        """
+        Verify IOs before and after control pod fails over after restarting node hosting control
+        pod.
+        """
+        LOGGER.info("STARTED: Verify IOs before and after control pod fails over after restarting"
+                    " node hosting control pod")
+
+        LOGGER.info("Step 1: Create IAM user and perform WRITEs-READs-Verify with "
+                    "variable object sizes. 0B + (1KB - 512MB)")
```

Comment:
remove the sizes "0B + (1KB - 512MB)" and for h/w it will differ, do changes at other places too

Reply:
We had decided to add sizes in logger.